doc/Optional-Feature-Control.md

```diff
@@ -0,0 +1,20 @@
+# SONiC Optional Feature Control Enhancement#
```

Comment:
Delete trailing `#` or add a space before it

Reply:
added space